.github/workflows/mocha.yml

```diff
@@ -22,25 +22,67 @@ jobs:
   smoke:
     uses: ./.github/workflows/npm-script.yml
     with:
-      os: 'ubuntu-latest,windows-2019'
-      node-versions: '14,16,18,20,22'
-      npm-script: test:smoke
+      node-versions: '14,22,lts/*'
+      npm-script: test-smoke
 
-  test-node:
+  test-node-lts:
```

Comment:
Did some work to speed up the tests by lessening the smoke tests so that we can head towards the parallel tests as fast as possible, and eg. windows tests are notoriously slow

Also added a full `test-node-lts` smoke test to get a full test through before starting with all the windows testing